js/download.js

```diff
@@ -190,3 +202,41 @@ function saveToDevice() {
   a.click();
   document.body.removeChild(a);
 }
+
+function bookmark() {
+  if (localStorage.getItem("favorites") == null) {
+    localStorage.setItem("favorites", "[]");
+  }
+
+  let img = localStorage.getItem("BG");
+  let quote = localStorage.getItem("quote");
+  let id = localStorage.getItem("id");
+
+  let list = JSON.parse(localStorage.getItem("favorites"));
+  let index = list.findIndex((item) => item.includes(id));
```

Comment:
When you already know the index of the element (id) of the subArray (item) just use it
like this `  let index = list.findIndex((item) => item[2]===id);`
In your case you have just 3 elements (indexes),
But imagine that your subArray contains 1000 or 1000,000 elements that's will affect the time complexity
Do the best practice even when it is not highly required